apps/registry/public/r/styles/tokens.css

```
@@ -0,0 +1,170 @@
+    alternate;
+  --animate-spin-around: spin-around calc(var(--speed) * 2) infinite linear;
+
+  /* MagicUI: animated-gradient-text */
+  --animate-gradient: gradient 8s linear infinite;
+
+  /* MagicUI: marquee */
+  --animate-marquee: marquee var(--duration) infinite linear;
+  --animate-marquee-vertical: marquee-vertical var(--duration) linear infinite;
+
+  /* Aceternity: background-gradient-animation */
+  --animate-first: moveVertical 30s ease infinite;
```

Stylelint (17.11.0)

[error] Expected "moveVertical" to be "movevertical" (value-keyword-case)

(value-keyword-case)